.husky/commit-msg

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+if ! (cat "$1" | grep -q "Signed-off-by:"); then
+  >&2 echo "Commit message must be signed off with your user name and email."
+  >&2 echo "To sign off your commit, add the -S flag to the git commit command."
```

Comment:
also, should this be a lower case `s`?  I get an error running with capital `S` but I usually sign off with lower case:
```
C:\dev\asm\make>git commit -S -m "test"
gpg: skipped "Dan Kelosky <dkelosky@gmail.com>": No secret key
gpg: signing failed: No secret key
error: gpg failed to sign the data
fatal: failed to write commit object
```

Versus:
```
C:\dev\asm\make>git commit -s -m "test" 
[master ad25c86] test
 1 file changed, 12 insertions(+), 5 deletions(-)
```

Reply:
Good catch! I assumed the options were case insensitive, but they definitely aren't based on your example.